source/features/clean-repo-filelist-actions.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'dom-chef';
+import onetime from 'onetime';
+import {observe} from 'selector-observer';
+import PlusIcon from 'octicon/plus.svg';
+import SearchIcon from 'octicon/search.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
+	// Only select desktop version of the button
+	observe('.d-flex > [data-hotkey="t"]:not(.rgh-clean-actions)', {
+		add(searchButton) {
+			searchButton.classList.add('tooltipped', 'tooltipped-ne', 'rgh-clean-actions');
+			searchButton.setAttribute('aria-label', 'Go to file');
+
+			searchButton.firstChild!.replaceWith(<SearchIcon/>);
+
+			// Exclude logged out, mobile or file pages
+			const addButtonText = searchButton.nextElementSibling?.querySelector('.d-md-flex');
+			if (addButtonText) {
+				addButtonText.parentElement!.classList.add('tooltipped', 'tooltipped-ne');
```

Comment:
👇 (if `parentElement` is a `<summary>`)

Reply:
In what case it's not a summary? My understanding is that it's a summary when it's a dropdown, and it always is.